### PR TITLE
Add rules for updating the CF data model

### PR DESCRIPTION
**Pull request by davidhassell**
PR for issue #71

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -36,3 +36,19 @@ title: Rules
...
+
+<p> All new proposals will be assessed to see if the new features defined in the proposal map onto the CF data model.
+
+<p> The assessment will be carried out by a member of the conventions  committee or another suitably qualified person. If no-one  volunteers, the chairman of the committee will ask someone to do it.
+
+<p> If the proposal maps onto the existing CF data model then no  modifications to the data model are required.
+
+<p> Otherwise an attempt must be made to modify the proposal so that its  new features do map onto the CF data model, and in such a way that  the proposal's intent is not compromised.
+
+<p> If the proposal cannot be acceptably modified to conform to the existing data model, then the data model will need to be modified to  accommodate the new features. This modification will be either backwards compatible or backwards incompatible. The former (preferable) case occurs if the data model may be extended or generalised in some way that allows the new features but does not affect its existing constructs and relationships.
```
Comment (erget) [inline rules.md:52]: Maybe strike
> This modification will be either backwards compatible or backwards incompatible.

and follow up with

> If the data model may be extended or generalised in some way that allows the new features but dose not affect its existing constructs and relationships, the proposal is considered backwards compatible. This is the preferred solution.

**Comment by davidhassell** [comment]
@erget I have no objection to removing the `<p>` tags. I merely copied the the style of the rest of the existing document, so I guess that they could go throughout if the markdown does not need them.

File: rules.md (hunk 1 above)
Comment (davidhassell) [reply rules.md:52]: Thanks - I'll update the PR with this.

**Comment by JonathanGregory** [comment]
This concludes https://github.com/cf-convention/cf-conventions/issues/159

**Pull request merged**